sdk/examples/cawg_identity.rs

```diff
@@ -125,9 +125,7 @@ mod cawg {
 
         builder.sign_file_async(&signer, source, &dest).await?;
 
-        let mut reader = Reader::from_file(dest)?;
-
-        reader.post_validate_async(&CawgValidator {}).await?;
+        let reader = Reader::from_file_async(dest).await?;
```

Comment:
Why not keep the sync interface too? I am not sure we can make everything async

Reply:
IMHO we shouldn't be hiding the fact that some APIs are inherently asynchronous. We can and should push the cost of bridging that gap as far up the API chain as possible. Leaving this as is.